doc/developer-notes.md

```diff
@@ -408,6 +408,21 @@ Source code organization
 
   - *Rationale*: Avoids symbol conflicts
 
+- Terminate namespaces with a comment (`// namespace mynamespace`). The comment
+  should be placed on the same line as the brace closing the namespace, e.g.
+
+  ```c++
```

Comment:
@jtimon Sorry I'm not sure I follow - where should the extra spaces be placed? :-)